README.md

```diff
@@ -10,20 +22,23 @@ Also, a `cancelAllSubscriptions` function is passed to the decorated component a
 no arguments, and will cancel all subscriptions that were registered via `cancelWhenUnmounted`. It will also reset the list of
 subscriptions to be empty, so that future calls to `cancelWhenUnmounted` and `cancelAllSubscriptions` will start fresh.
 
-# Installation
-`npm install react-disposable-decorator`
+## `CancelableElement` - Similar to the decorator, but renders children passing both `cancelWhenUnmounted` and `cancelAllSubscriptions` as parameters (see example)
```

Comment:
Honestly I don't see any case where this way of using react-cancelable is better than the decorator way, since I can't image a situation where `cancelWhenUnmounted` and `cancelAllSubscription` are used inside of the JSX. I think this is different than a component like `<FetchUser>` (from the Ryan Florence talk on this) because the props passed to CancelableElement's "function as a child" are functions (instead of a "user" object) that aren't really used in the JSX. And having to set `this.cancelWhenUnmounted = cancelWhenUnmounted` inside of the "function as a child" is a side effect of a jsx-y / rendering function, and will get executed over and over again ever render flow even though the cancelWhenUnmounted and cancelAllSubscriptions functions aren't changing from render to render. Using this pattern also makes the render tree less intuitive because the children of CancelableElement are created by the parent component but rendered by the CancelableElement component.

Anyway, I think this pattern is overall a pretty cool pattern but don't know if it's very useful for this particular situation. Won't hurt to keep it here, but not sure if it helps, either.